### PR TITLE
JNDI-ify MQ connection information

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java [hunk 1]
```diff
@@ -32,6 +41,10 @@
 
 	private static final Logger log = LoggerFactory.getLogger(MicroserviceMsgservice.class);
 
+	public final static String MQ_HOST_JNDI_NAME = "messaging.service.host";
```
Comment (jiwalker-usgs) [inline src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java]: Shouldn't this come from service discovery?

File: src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java [hunk 2]
```diff
@@ -32,6 +41,10 @@
...
+	public final static String MQ_USER_JNDI_NAME = "messaging.service.user";
```
Comment (jiwalker-usgs) [inline src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java]: And we really need to find somewhere more central to put the user and pass as well.

File: src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java (hunk 1 above)
Comment (thongsav-usgs) [inline src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java]: yep (although not sure how to supply user/pass). I'm working on two tickets right now (messaging and service discovery). Trying to wrap up the messaging before moving onto the service discovery ticket.

Comment (jiwalker-usgs) [inline src/main/java/gov/usgs/cida/microservices/messaging/MicroserviceMsgservice.java]: That's fair, just want to make sure we aren't going to force these JNDI settings on any user of the microservices-libraries.

**Pull request merged**